tools/vault/config/init.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+export VAULT_ADDR='https://vault.apps.digilab.network'
+
+# Initialize Vault and capture the output
+INIT_OUTPUT=$(vault operator init -key-shares=8 -key-threshold=2)
+
+# Extract the unseal keys and root token from the output
+UNSEAL_KEYS=$(echo "$INIT_OUTPUT" | grep 'Unseal Key' | awk '{print $NF}')
+ROOT_TOKEN=$(echo "$INIT_OUTPUT" | grep 'Initial Root Token' | awk '{print $NF}')
+
+echo "Root token: $ROOT_TOKEN"
+
+echo "Unseal keys: $UNSEAL_KEYS"
```

Comment:
It feels a bit tricky to log a token?

Reply:
je hebt geen keuze. als je geen root token of unseal keus hebt kan je vault niet gebruiken. deze scripts zijn bedoeld voor handmatige executie. Heb ze hier ingezet zodat iemand de deployment zou kunnen reproduceren